Dockerfile

```diff
@@ -1,10 +1,20 @@
 FROM python:3.5.3-alpine
 MAINTAINER paul.traylor@linecorp.com
 
+ENV PROMETHEUS_VERSION 2.1.0
```

Comment:
> Note: This PR contains the Prometheus version hard-coded in the Dockerfile. Alternatively, the latest version could be fetched using the Github API.

I think hardcoding this is not a problem. Ultimately I added this to help check syntax errors and such before deploying to Prometheus so it likely does not always need to be the absolute latest version.